server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts

```diff
@@ -71,6 +72,35 @@ investigationInfo.get('/staticInfo', (request: Request, response: Response) => {
         });
 });
 
+investigationInfo.get('/groupedInvestigations/reasons', adminMiddleWare,  (request: Request, response: Response) => {
+    const reasonsLogger = logger.setup({
+        workflow: 'query reasons for grouped investigations',
+        user: response.locals.user.id,
+        investigation: response.locals.epidemiologynumber
+    })
+    reasonsLogger.info('requesting the graphql API to query reasons', Severity.LOW);
+    const reasons = [
```

Comment:
why didnt you query from DB

Reply:
I'm waiting for naaman to do it. I'll change it when he's done

Reply:
😲😲😲